back/src/domains/establishment/adapters/PgDiscussionRepository.ts

```diff
@@ -122,12 +123,12 @@ export class PgDiscussionAggregateRepository
     return (
       discussion && {
         ...discussion,
-        createdAt: new Date(discussion.createdAt),
+        createdAt: new Date(discussion.createdAt).toISOString(),
         immersionObjective: discussion.immersionObjective ?? null,
         exchanges: discussion.exchanges
-          ? discussion.exchanges.map(({ sentAt, ...rest }: ExchangeEntity) => ({
-              sentAt: new Date(sentAt),
-              ...rest,
+          ? discussion.exchanges.map((exchange) => ({
+              ...exchange,
+              sentAt: new Date(exchange.sentAt).toISOString(),
```

Comment:
`sentAt` est déjà un DateString

```suggestion
              sentAt: exchange.sentAt,
```